go/src/koding/kites/kloud/main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer k.Close()
```

Comment:
I think socialapi suffers from the same problem, all the defers here won't get executed e.g. on sigterm:

https://github.com/koding/koding/blob/master/go/src/socialapi/workers/api/main.go#L38